int32[] indices
int32 type
------------------
std_msgs/Bool ifdone
